doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -453,14 +453,16 @@ Matter samples
 
   * The :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILES_PATH` Kconfig option, which specifies ZAP files location for the sample.
     By default, the option points to the :file:`src/default_zap` directory and can be changed to any path relative to sample's location that contains the ZAP file and :file:`zap-generated` directory.
-  * Support for :ref:`Trusted Firmware-M <ug_tfm>` on the nRF54L15 PDK.
+  * Support for nRF54L15 DK.
+  * Support for :ref:`Trusted Firmware-M <ug_tfm>` on the nRF54L15 DK.
```

Comment:
invalid statement based on information from stand-up

Reply:
I will change it to SoC because actually, it works on DK with FW built for PDK. I've added such a comment to the samples' readme.